mfecfe/entidades.py

```diff
@@ -216,8 +216,6 @@
 import re
 import xml.etree.ElementTree as ET
 
-from decimal import Decimal
```

Comment:
Se vc remover este import vc vai ferrar
if not isinstance(value, Decimal) and not isinstance(value, D):